cmd/run.go

```diff
@@ -462,23 +474,15 @@ func RunServer(ctx context.Context, config *Config) error {
 		return fmt.Errorf("failed to initialize authenticator: %w", err)
 	}
 
-	unaryServerInterceptors := []grpc.UnaryServerInterceptor{
-		grpc_validator.UnaryServerInterceptor(),
-		grpc_auth.UnaryServerInterceptor(middleware.AuthFunc(authenticator)),
-		middleware.NewRequestIDInterceptor(logger),
-		middleware.NewLoggingInterceptor(logger),
-	}
-
-	streamingServerInterceptors := []grpc.StreamServerInterceptor{
-		grpc_validator.StreamServerInterceptor(),
-		grpc_auth.StreamServerInterceptor(middleware.AuthFunc(authenticator)),
-		middleware.NewStreamingRequestIDInterceptor(logger),
-		middleware.NewStreamingLoggingInterceptor(logger),
-	}
+	unaryInterceptors, streamingInterceptors := middleware.NewInterceptors(&middleware.Config{
```

Comment:
Perhaps this should have been in a different PR.

Reply:
Different one it is. Thanks!